src/features/create-quiz/store/CreateQuizStore.tsx

```diff
@@ -0,0 +1,51 @@
+import { makeAutoObservable, toJS } from "mobx";
+
+export interface QuestionItem {
+  id: number;
+  title: string;
+  completed: boolean;
+}
+
+export class QuizStoreImpl {
+  questions: QuestionItem[] = [];
+
+  constructor() {
+    makeAutoObservable(this, {}, { autoBind: true });
+  }
+
+  addQuestion(title: string) {
+    const item: QuestionItem = {
+      id: +Math.random().toFixed(4),
```

Comment:
For ids generation you could use react-uuid, which might be a more reliable method to generate unique ids.